src/tuvx/interface_util.F90

```diff
@@ -312,6 +323,40 @@ function to_c_string( f_string ) result( c_string )
 
   end function to_c_string
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Creates a new string_t_c from a fortran character array
+  function create_string_t_c( f_string ) result( c_string )
+    use iso_c_binding,                 only : c_char, c_null_char, c_loc
```

Comment:
It doesn't look like using any of those types. Does it make sense to remove them? Additionally, for consistency with the other function comments, should we remove the 's' in 'Creates'?

Reply:
removed!